src/Runtime.ts

```diff
@@ -130,10 +130,6 @@ export function evaluateComparison(operator: string, left: any, right: any): boo
                 return comparison >= 0;
             case '>':
                 return comparison > 0;
-            case '==':
-                return comparison === 0;
-            case '!=':
-                return comparison !== 0;
```

Comment:
I actually think you can keep all of this in this file. CEL under the hood for strings just do the size. So you can do something like this in CEL and that is valid
`'Yesa' > 'Yes'` => would evaluate to true.

You can try it out here
https://playcel.undistro.io/